server/build/webpack.js

```diff
@@ -10,6 +10,12 @@ import WatchRemoveEventPlugin from './plugins/watch-remove-event-plugin'
 import DynamicEntryPlugin from './plugins/dynamic-entry-plugin'
 import DetachPlugin from './plugins/detach-plugin'
 
+// get all the environment variables
+const env = Object.keys(process.env).reduce(function (o, k) {
```

Comment:
I agree with the prefix, otherwise someone might expose something they didn't mean to.